apps/super-app/src/App.tsx

```diff
@@ -1,17 +1,28 @@
 import './App.css';
+import { Route, Routes } from "react-router-dom";
 import { ThemeProvider } from './common/contexts';
 import {
   PrimaryButton,
   NeutralButton,
   ErrorButton,
 } from './common/components/RoundedButton';
+import CallHome from "./common/components/CallHome";
+import Support from "./common/components/Support";
+import More from "./common/components/More";
+import NavBar from "./common/components/navbar/Navbar";
 
 function App() {
   return (
     <ThemeProvider>
       <PrimaryButton>Primary</PrimaryButton>
       <NeutralButton>Neutral</NeutralButton>
       <ErrorButton>Error</ErrorButton>
+      <NavBar/>
+      <Routes>
+          <Route path="/callhome" element={<CallHome />}></Route>
+          <Route path="/support" element={<Support />}></Route>
+          <Route path="/more" element={<More />}></Route>
+        </Routes>
```

Comment:
Oh neat, this actually uses the React router state to determine the active state. I originally envisioned that the nav bar would just use internal state, but this is okay too (TIL about NavLink, good find!). I didn't intend for you to add React router because this would conflict quite drastically with https://github.com/bettersg/call-home/pull/102, which also adds React router. I _think_ there's no need to change the structure of this PR, but FYI I will tweak this  and https://github.com/bettersg/call-home/pull/102 to be more consistent.